packages/web-console/serve-dist.js

```diff
@@ -1,83 +0,0 @@
-const http = require("http")
-const url = require("url")
-const fs = require("fs")
-const path = require("path")
-
-const contextPath = process.env.QDB_HTTP_CONTEXT_WEB_CONSOLE || ""
-
-const server = http.createServer((req, res) => {
-    if (path.normalize(decodeURI(req.url)) !== decodeURI(req.url)) {
-        res.statusCode = 403;
-        res.end();
-        return;
-    }
-  const { method } = req
-  const baseUrl =  "http://" + req.headers.host + contextPath;
-  const reqUrl = new url.URL(req.url, baseUrl);
-  const reqPath = reqUrl.pathname + reqUrl.search;
-  const reqPathName = reqUrl.pathname;
-
-  if (
-    reqPathName.startsWith("/exec") ||
-    reqPathName.startsWith("/settings") ||
-    reqPathName.startsWith("/warnings") ||
-    reqPathName.startsWith("/chk") ||
-    reqPathName.startsWith("/imp") ||
-    reqPathName.startsWith("/exp")
-  ) {
-    // proxy /exec requests to localhost:9000
-    const options = {
-      hostname: "localhost",
-      port: 9000,
-      path: contextPath + reqPath,
-      method,
-      headers: req.headers,
-    }
-
-    const proxyReq = http.request(options, (proxyRes) => {
-      res.writeHead(proxyRes.statusCode, proxyRes.headers)
-      proxyRes.pipe(res, { end: true })
-    })
-
-    req.pipe(proxyReq, { end: true })
-  } else if (
-    reqPathName.startsWith("/userinfo")
```

Comment:
Many enterprise tests are failing, I think because vite config does not have an equivalent of the code from `serve-dist.js` which mocked the `/userinfo` endpoint of the OAuth2 provider.